create3_lidar/README.md

```diff
@@ -0,0 +1,71 @@
+# iRobot® Create® 3 LIDAR SLAM demo
+
+This example sets up LIDAR SLAM with a Create® 3 robot and Slamtec RPLIDAR spinning laser rangefinder.
```

Comment:
Do we want a picture showing what to expect after the tutorial is successfully run

Reply:
Yes; do you have one we can use? I can put it wherever I put the other one.